tests/test_integration.py

```diff
@@ -246,6 +246,13 @@ def fingerprint(self,
                 2.14015989,
                 ['--features_path', os.path.join(TEST_DATA_DIR, 'regression.npz'), '--no_features_scaling']
         ),
+        (
+                'chemprop_features_generator_features_path',
+                'chemprop',
+                'rmse',
+                1.59283050,
+                ['--features_generator', 'morgan', '--features_path', os.path.join(TEST_DATA_DIR, 'regression.npz'), '--no_features_scaling']
+        ),
```

Comment:
What is the behavior of `--no_features_scaling` when both loaded features and feature generators are applied? Is `--no_features_scaling` applied to both features? If so, in the Readme, we recommended `--no_features_scaling` for `rdkit_2d_normalized`, but if this flag is included while user is providing loaded features, does that mean all features will not be normalized? (probably not the desired behavior since we normally do want to scale loaded features).

Reply:
You are right! In this scenario, I would suggest using `rdkit_2d` as the generator instead of `rdkit_2d_normalized`. How about adding a description in the README.md to mention that if both loaded features and feature generators of `rdkit_2d_normalized` are applied, and the loaded features need to be normalized, `rdkit_2d_normalized` should be replaced with `rdkit_2d`.

Reply:
Thanks. Please add a description.